src/github/pullRequestManager.ts

```diff
@@ -14,7 +14,8 @@ import { IPullRequestManager, IPullRequestModel, IPullRequestsPagingOptions, PRT
 import { PullRequestGitHelper } from "./pullRequestGitHelper";
 import { PullRequestModel } from "./pullRequestModel";
 import { parserCommentDiffHunk } from "../common/diffHunk";
-import { Configuration } from "../configuration";
+import { Configuration } from '../authentication/configuration';
+import { GitHubManager } from '../authentication/githubserver';

```

Comment:
`../authentication/githubServer`

Reply:
ooof, good catch, that would fail on linux :/